observatory_platform/dags/geonames.py

```diff
@@ -17,16 +17,16 @@
 from datetime import datetime
 
 from airflow import DAG
-from airflow.operators.python_operator import PythonOperator
+from airflow.operators.python_operator import PythonOperator, ShortCircuitOperator
 
 from observatory_platform.telescopes.geonames import GeonamesTelescope
 
 default_args = {
     "owner": "Airflow",
-    "start_date": datetime(2020, 6, 1)
+    "start_date": datetime(2020, 9, 1)
 }
 
-with DAG(dag_id="geonames", schedule_interval="@monthly", default_args=default_args, catchup=False) as dag:
+with DAG(dag_id="geonames", schedule_interval="@weekly", default_args=default_args, catchup=False) as dag:
```

Comment:
Thanks, that's good to know. I changed this to weekly to make it easy to use external task sensors. There is an external task sensor linked to each of the telescope DAGs (see [dags/doi.py](https://github.com/The-Academic-Observatory/observatory-platform/blob/feature-aggregation-workflow/observatory_platform/dags/doi.py#L48-L83)), when their state is set to success, then the doi DAG runs. 

I don't know how to make an external task sensor automatically go into a success state when a DAG isn't scheduled for a day or when it isn't the first Sunday of the month?

Reply:
Ah, I missed that the external task sensors are dependent on the execution date. Are you just interested in that the latest states are successful?

Perhaps it gives some more flexibility to get the dag ids, and check their state just like we're doing with the vm_create/destroy.

Reply:
Thanks. I would ideally like to to keep using the ExternalTaskSensors because they are simple and there isn't much to go wrong with them and it enables the relationships between the DAGs that the DOI DAG is dependent on explicit and somewhat visible in the Airflow UI.